### DISCUSSION TITLE
[JOB OFFER] [ADDON] Chardump Wotlk to other versions

### DISCUSSION THREAD

**Discussion by Naxxramas-Server** [azerothcore/azerothcore-wotlk#3944] [Job offers]
Hello all , i'm searching for someone who can adapt a wotlk chardump addon to make it compatible with other client version (1.12 , 2.4.3, and highter versions).

I can share the addon and add bounty if someone is interestef by this work.

**Comment by Naxxramas-Server** [comment]
Bump , if someone interested , paid work !

**Comment by Naxxramas-Server** [comment]
Up , still searching for someone

**Comment by Naxxramas-Server** [comment]
Up , still searching for this work

**Comment by Naxxramas-Server** [comment]
Up

**Comment by Kaev** [comment]
What exactly doesn't work? Is it just the lua addon that needs rework or is there also some SQL work to do?
How much do you pay (e.g. per version?)?
And can you provide a server for each version so i can test it?
I could offer to do it with vanilla, TBC and WotLk. Don't really wanna download the other, newer clients.
